astroid/protocols.py

```diff
@@ -595,7 +639,12 @@ def __enter__(self):
 
 
 @decorators.raise_if_nothing_inferred
-def named_expr_assigned_stmts(self, node, context=None, assign_path=None):
+def named_expr_assigned_stmts(
+    self: nodes.NamedExpr,
+    node: mixins.AssignTypeMixin,
```

Comment:
It has been awhile since I last looked at that. Is this right?

Reply:
Incorrect indeed, fixed!